src/reflect/scala/reflect/internal/Flags.scala

```diff
@@ -169,6 +169,11 @@ class Flags extends ModifierFlags {
 
   final val SYNCHRONIZED  = 1L << 45      // symbol is a method which should be marked ACC_SYNCHRONIZED
 
+  final val SYNTHESIZE_IMPL_IN_SUBCLASS = 1L << 50 // used in fields phase to indicate this accessor should receive an implementation in a subclass
+
+  // flags used strictly internally in the Fields phase (info/tree transform):
+  final val NEEDS_TREES   = 1L << 59           // this symbol needs a tree. (distinct from SYNTHESIZE_IMPL_IN_SUBCLASS)
+
```

Comment:
Symbols with the NEEDS_TREES flag could be tracked in a set (I opted not to because I assumed the flag would be more efficient). The SYNTHESIZE_IMPL_IN_SUBCLASS flag will replace lateDEFFERRED in Fields: the Sequel, so that one needs to remain a flag.